weather_mv/loader_pipeline/ee.py

```diff
@@ -376,8 +376,8 @@ def expand(self, paths):
                 | 'IngestIntoEE' >> IngestIntoEETransform.from_kwargs(**vars(self))
             )
 
-            if self.use_metrics:
-                output | 'AddMetrics' >> beam.ParDo(AddMetrics())
+            if self.use_metrics and not self.skip_region_validation:
```

Comment:
Yes.

Reply:
It is still weird because depending upon `skip_region_validation` at this point doesn't seem right. Tracking this in the future if skip_region_validation changes would be difficult. @mahrsee thoughts?

Reply:
Additionally, we could introduce two new flags namely: `--use_beam_metrics` & `--use_monitoring_metrics` and change the pipeline flow accordingly.

Reply:
Definitely lets have a separate flag. `--use_monitoring_metrics` sg! 

Let's add beam_metrics to all the pipelines.